releases/v21.2.6.md

```diff
@@ -0,0 +1,74 @@
+- S3 URIs used in [`BACKUP`](../v21.2/backup.html), [`EXPORT`](../v21.2/export.html), or [`CREATE CHANGEFEED`](../v21.2/create-changefeed.html) statements can now include the query parameter `S3_STORAGE_CLASS` to configure the storage class used when that job creates objects in the designated S3 bucket. [#75608][#75608]
+- Non-admin users can now use the [`SHOW RANGES`](../v21.2/show-ranges.html) statement if the [`ZONECONFIG`](../v21.2/grant.html#supported-privileges) privilege is granted. [#76071][#76071]
+- `ST_MakePolygon` is now disallowed from making empty polygons from empty linestrings. This is not allowed in PostGIS. [#76255][#76255]
+
+### Bug fixes
+
+- A sequence is created when a column is defined as [`SERIAL`](../v21.2/serial.html) type and the `serial_normalization` session variable is set to `sql_sequence`. In this case, the sequence is owned by the column and the table where the column exists. The sequence should be dropped when the owner table/column is dropped, which is the PostgreSQL behavior. However, CockroachDB never set ownership information correctly but only dependency relationship. Thus, the sequence stays even the owner table/column does not exist anymore. This fix assigns correct ownership information to sequence descriptor and column descriptor to align with PostgreSQL behavior. [#75704][#75704]
+- Fixed a bug that could cause nodes to crash when truncating abnormally large Raft logs. [#75979][#75979]
+- The [DB Console](../v21.2/ui-overview.html) [Databases](../v21.2/ui-databases-page.html) page now shows stable, consistent values for database sizes. [#76324][#76324]
+
+### Performance improvements
+
+- Sorting data with bytes-like types ([`BYTES`](../v21.2/bytes.html), [`STRING`](../v21.2/string.html), [`JSON`/`JSONB`](../v21.2/jsonb.html), [`UUID`](../v21.2/uuid.html)) when [`LIMIT`](../v21.2/limit-offset.html) clause is specified has now become more predictable. [#75847][#75847]
+
+### Contributors
+
+This release includes 25 merged PRs by 21 authors.
+
+[#75608]: https://github.com/cockroachdb/cockroach/pull/75608
+[#73024]: https://github.com/cockroachdb/cockroach/issues/73024
+[#75704]: https://github.com/cockroachdb/cockroach/pull/75704
+[#75847]: https://github.com/cockroachdb/cockroach/pull/75847
+[#75858]: https://github.com/cockroachdb/cockroach/pull/75858
+[#75893]: https://github.com/cockroachdb/cockroach/pull/75893
+[#75979]: https://github.com/cockroachdb/cockroach/pull/75979
+[#76071]: https://github.com/cockroachdb/cockroach/pull/76071
+[#76250]: https://github.com/cockroachdb/cockroach/pull/76250
+[#76255]: https://github.com/cockroachdb/cockroach/pull/76255
+[#76321]: https://github.com/cockroachdb/cockroach/pull/76321
+[#76324]: https://github.com/cockroachdb/cockroach/pull/76324
+[64156f955]: https://github.com/cockroachdb/cockroach/commit/64156f955
```

Comment:
Are we linking to these commits somewhere? If so, we should change those links to point to the parent PR and then remove these.

Reply:
This style of link is known as a reference link. The syntax of calling it in-line is [#76255][#76255], and the references you see on lines 70 and below are hidden by default. The script has been generating links in this manner prior to my changes.

Reply:
Yep, understood. I'm referring to lines 73 and 74. We can remove those.

Reply:
Removed the loose commit lines. Thanks!